canary/apps/angular/angularcli/package.json

```diff
@@ -20,8 +20,8 @@
     "@angular/platform-browser": "~11.2.14",
     "@angular/platform-browser-dynamic": "~11.2.14",
     "@angular/router": "~11.2.14",
-    "@aws-amplify/ui-angular": "latest",
-    "aws-amplify": "latest",
+    "@aws-amplify/ui-angular": "v3-latest",
```

Comment:
Updated to v3-latest. This isn't published yet, but will be hotfix is done. Note that CI will change to `v3-next` during build tests, so build tests aren't blocked.